libafl/src/executors/hooks/unix.rs

```diff
@@ -51,12 +51,6 @@ pub mod unix_signal_handler {
         ) {
             unsafe {
                 let data = &raw mut GLOBAL_STATE;
-                let in_handler = (*data).set_in_handler(true);
-
-                if in_handler {
-                    log::error!("We crashed inside a crash handler, but this should never happen!");
-                    libc::exit(56);
```

Comment:
we should make this constant?

const EXITCODE_TRIPLE_CRASH: usize = 56;

Reply:
Also, why 56 and not something fun like 42

Reply:
idk. it's a totally random number. no specific meanings